libp2p/peer.nim

```diff
@@ -8,10 +8,15 @@
 ## those terms.
 
 ## This module implementes API for libp2p peer.
+
```

Comment:
this module looks like it should be called `peerid` - wonder if it's possible to move the peerid type in here as well since it should be standalone

Reply:
you mean `PeerInfo`? cos `PeerID` is here

Reply:
the `type` section is not in here

Reply:
hmm? this one https://github.com/status-im/nim-libp2p/blob/c788a6a3c0ca614be4c67cc566aee6877da0250b/libp2p/peer.nim#L22 ?

Reply:
oops, I don't know how I missed it - clearly it's right where it should be :)